roles/download/defaults/main.yml

```diff
@@ -23,6 +23,9 @@ download_always_pull: False
 # Use the first kube-master if download_localhost is not set
 download_delegate: "{% if download_localhost %}localhost{% else %}{{groups['kube-master'][0]}}{% endif %}"
 
+# Arch of Docker images and needed packages
```

Comment:
This is not how defaults works. Just put it like this:

# Arch of Docker images and needed packages
image_arch: amd64

There's no need to quote if it starts with a letter.